config/routes.rb

```diff
@@ -33,6 +33,9 @@
       end
     end
 
+    resources :pdf_templates
+    get '/sites/:district_code/pdf_templates/:id/preview', to: 'sites/pdf_templates#show', defaults: { format: "pdf" }
```

Comment:
@radinreth please update url to use:
    1. use params :site_code instead of :district_code
    2. use /:id.pdf instead of /:id/preview

Reply:
noted bong